test/e2e/util.go

```diff
@@ -3351,3 +3352,89 @@ func isElementOf(podUID types.UID, pods *api.PodList) bool {
 	}
 	return false
 }
+
+// Retrieve a list of schedulable nodes whose condition is Ready.  An
```

Comment:
minor nit: the function doc comments [should start with the name of the function](https://golang.org/doc/effective_go.html#commentary), i.e.
```
// GetReadyNodes retrieves a list of schedulable nodes ...
```